code/script.js

```diff
@@ -1,51 +1,312 @@
   chat.scrollTop = chat.scrollHeight;
+  messageSound();
 }
 
-// Starts here
+
+// Conversations starts here
+
+// Question 1 - Bot asks 'What is your room number'?
+
 const greeting = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+      showMessage("Welcome to Hotel Vosje. What is your room number?", 'bot'),2000; 
+    };
+  
+
+// User answers with room number
+
+const handleFormInput = (event) => {        // at submit this function will be invoked
+  event.preventDefault();        // prevents website refresh at submit
+  const room = roomInput.value;        // input value will be stored in the const name
+  showMessage(`My room number is ${room}`, 'user');      // users answer 
+  roomInput.value = '';        // clearing room input setting it to an empty string
+
+  if (room === '') {       // this will prompt the user to answer correctly
+    showMessage(`Please, give me your room number.`, 'bot');
+  } else {
+    greetingLoop(room);
+  }
+}
+
+// The bot answers hello room #. This const will also be used as loop back selections if guest needs something else
+
+const greetingLoop = (room) => {
+  typing(); 
+  setTimeout(() => removeTyping(), 600)
+  showMessage(`Hello room ${room}`, 'bot');
+  setTimeout(() => helpSelection(), 600)
+}
+
+// Question 2 - Bot asks 'What do you need help with today?' and sends selection-buttons
+
+const helpSelection = () => {
+  showMessage('What can I help you with?', 'bot');
+  inputWrapper.innerHTML = `
+  <button id="amenities" type="sbumit" class="chat-btn">Amenities </button>
+  <button id="room-service" type="submit" class="chat-btn">Room service </button>
+  <button id="wake-up" type="submit" class="chat-btn">Wake-up call </button>
+  `
+  document.getElementById('amenities').addEventListener('click',() => {
+    showMessage(`I need amenities, please`, 'user')
+    setTimeout( ()=> {
+      preloader.style.display = "none";
+     }, 2000);
+    setTimeout(() => selectionAnswer('amenities'),1500);
+})
+
+  document.getElementById('room-service').addEventListener('click',() => {
+    showMessage(`I need food, please`, 'user');
+    setTimeout(() => selectionAnswer('room-service'),1500);
+})
+
+  document.getElementById('wake-up').addEventListener('click',() => {
+    showMessage(`I need a wake-up-call.`, 'user')
+    setTimeout(() => selectionAnswer('wake-up'),1500);
+  })
+
+}
+
+// Question 3 - Bot asks what guest needs depending on what guest chose in helpselection
+
+const selectionAnswer = (selection) => {
+
+  if (selection === 'amenities') {         // If guest needs amenities 
+    showMessage(`Sure, what do you need?`, 'bot');
+    inputWrapper.innerHTML = `
```

Comment:
The page scrolling does not show a message properly after choosing amenities. The buttons are bigger and overlap the main window.

Reply:
ok thanks we will look at that and try to fix it!